README.md

```diff
@@ -1,15 +1,27 @@
 
-# (🚧 **WIP** 🚧) Fivetran Streamlit Billing Model
+# Fivetran Streamlit Billing Model
 
 ## 📣 Overview
+This Streamlit app showcases a variety of example reports, using simulated data from a fictional office supply company, to demonstrate the types of analyses possible within the Billing domain. These reports are built using different combinations of Fivetran connectors and data models, providing versatile examples of how to leverage Fivetran for billing analytics.
 
-## 📈 Provided reports
+Supported connector and data model combinations include:
 
-## 🎯 How do I use this Streamlit app?
+- [Stripe connector](https://fivetran.com/docs/connectors/applications/stripe) + [Stripe data model](https://fivetran.com/docs/transformations/data-models/stripe-data-model/stripe-transform-model)
+- [Shopify connector](https://fivetran.com/docs/connectors/applications/shopify) + [Shopify data model](https://fivetran.com/docs/transformations/data-models/shopify-data-model/shopify-transform-model)
+- [Recurly connector](https://fivetran.com/docs/connectors/applications/recurly) + [Recurly data model](https://fivetran.com/docs/transformations/data-models/recurly-data-model/recurly-transform-model)
+- [Recharge connector](https://fivetran.com/docs/connectors/applications/recharge) + [Recharge data model](https://fivetran.com/docs/transformations/data-models/recharge-data-model/recharge-transform-model)
+- [Zuora connector](https://fivetran.com/docs/connectors/applications/zuora) + [Zuora data model](https://fivetran.com/docs/transformations/data-models/zuora-data-model/zuora-transform-model)
 
-## Installaing Required Dependencies
+Each data model provides a standardized, denormalized `*__line_item_enhanced` table that ensures consistent reporting across platforms. This means that regardless of your company's billing platform, these reports can be easily replicated using the unified `*__line_item_enhanced` table.
```

Comment:
I think we could emphasize "platform- agnostic" in here. maybe we can swap out `unified` for that

Reply:
I really like that suggestion. Updating now!